R/tm_g_distribution.R

```diff
@@ -1270,7 +1306,7 @@ srv_distribution <- function(id,
 
     teal.widgets::verbatim_popup_srv(
       id = "rcode",
-      verbatim_content = reactive(teal.code::get_code(output_q())),
+      verbatim_content = reactive(teal.code::get_code(req(decorated_output_q()))),
```

Comment:
`decorated_output_q` is not created anywhere

Reply:
Should be fixed with last commits